phalcon/validation/validator/uniqueness.zep

```diff
@@ -185,52 +186,97 @@ class Uniqueness extends CombinedFieldsValidator
 				}
 			}
 
-			let attribute = this->getColumnNameReal(record, this->getOption("attribute", singleField)),
-				except = this->getOption("except");
+			let isModel = record instanceof Model;
+
+			if !(isModel || record instanceof Collection) {
+				throw new Exception("The uniqueness validator works only with Phalcon\\Mvc\\Model or Phalcon\\Mvc\\Collection");
+			}
+
+			if isModel {
+				let attribute = this->getColumnNameReal(record, this->getOption("attribute", singleField));
```

Comment:
I would like to avoid constructions like `this->method(this->method())`.
Could you please use local variable to avoid performance regression